### PR TITLE
Added div for supporter logos + pdxcg logo

### PR DESCRIPTION
**Pull request by SheriDover**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -87,6 +87,14 @@
       <div class="medal">
         <a href='https://operationcode.org'><img src="images/medal_navy.png" alt="opcodelogo"></a>
       </div>
+      <ul class="supporters">
```
Comment (rickr) [inline index.html]: Can you move this above the `Support` and below `Interested in learning to code?`

**Review by rickr — CHANGES_REQUESTED** [review]
Hi Sheri! Thanks for the PR. One small change - I'd like to get your logo up higher on the page.

One other question - does the image size need to be set or does it look OK?

File: index.html (hunk 1 above)
Comment (SheriDover) [reply index.html]: I'm happy to make changes - what exactly do you want above Support and below Interested in learning?

Comment (SheriDover) [reply index.html]: I think we have what you wanted : ) Let me know if you would like more or different changes

Comment (rickr) [reply index.html]: Whoops - sorry for being unclear! 

I'd like to take the whole 'supporters' section you're submitting and place it above the medal class. Consider that as our footer.

The order of the page should be:
* How can I help?
* Interested in learning to code?
* Supporters (your entire section)
* Support
* OC Medal (aka footer)

File: index.html [hunk 2]
```diff
@@ -83,10 +86,16 @@
 
         <p>Together we are Operation Code.</p>
       </p>
-
-      <div class="medal">
-        <a href='https://operationcode.org'><img src="images/medal_navy.png" alt="opcodelogo"></a>
-      </div>
+      <h1>Supporters</h1>
```
Comment (rickr) [inline index.html]: This should be an `<h2>` to match our other sections as well.

File: index.html (hunk 1 above)
Comment (SheriDover) [reply index.html]: You asked me about an image size. The OC image? It is a little blurry. You can resize it, or make it an SVG.

**Review by rickr — APPROVED** [review]

**Comment by rickr** [comment]
Looks good. Thanks again for the PR and the quick turnaround on the changes!

**Comment by SheriDover** [comment]
Rick, 

This was my first contribution! I'm so excited. The landing page is pretty simple, please let me know if I can help with other changes.

Cheers,
Sheri

**Comment by rickr** [comment]
Could have fooled me! Great job! I love seeing first commits.

I'd keep an eye on https://github.com/OperationCode/vettec/issues. There should be a few issues coming in in the near future. 

Right now we've got https://github.com/OperationCode/vettec/issues/4. If you're interested I can reserve it for you.

**Pull request merged**